src/mumble/ManualPlugin.cpp

```diff
@@ -45,7 +45,17 @@ Manual::Manual(QWidget *p) : QDialog(p) {
 	qgvPosition->scale(1.0f, 1.0f);
 	qgvPosition->setRenderHints(QPainter::Antialiasing);
 	qgsScene = new QGraphicsScene(QRectF(-5.0f, -5.0f, 10.0f, 10.0f), this);
-	qgiPosition = qgsScene->addEllipse(QRectF(-0.5f, -0.5f, 1.0f, 1.0f), QApplication::palette().text().color(), QBrush(Qt::red));
+
+	const float indicatorDiameter = 4.0f;
+	QPainterPath indicator;
+	// The center of the indicator's circle will represent the current position
+	indicator.addEllipse(QRectF(-indicatorDiameter / 2, -indicatorDiameter / 2, indicatorDiameter, indicatorDiameter));
+	// A line will indicate the indicator's orientation (azimuth)
+	indicator.moveTo(0, indicatorDiameter / 2);
+	indicator.lineTo(0,  indicatorDiameter);
+
+	qgiPosition = qgsScene->addPath(indicator);
+	// qgiPosition = qgsScene->addEllipse(QRectF(-0.5f, -0.5f, 1.0f, 1.0f), QApplication::palette().text().color(), QBrush(Qt::red));
```

Comment:
I would remove this line, we have Git's history in case we need it.

Reply:
Whoops - my intention was to remove it but apparently I forgot ^^

Reply:
No worries.

Reply:
Done